examples/operators/fft2d/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG BASE_IMAGE=oremda/oremda
+
+FROM ${BASE_IMAGE}
+
+COPY run.py /
+
+LABEL oremda.name="fft2d" \
+      oremda.ports.input.in.type="data" \
+      oremda.ports.output.real.type="data" \
+      oremda.ports.output.imag.type="data"
```

Comment:
Eventually we are going to need to be more specific in the these types, "data" is not going to be enough to know how a pipeline can be wired up.